### PR TITLE
proxyAuth to pass proxy authentication

### PR DESCRIPTION
**Pull request by MiShibari**
see title. Already implemented several times in SalesforceEng/global-tunnel branches, but never was commited to head. Thank you, @wzrdtales for creating a hard branch and maintance of this package. Worked pretty well for my cases.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "global-tunnel-ng",
-  "version": "1.2.1",
+  "version": "1.2.2",
```
Comment (wzrdtales) [inline package.json]: Can you please remove the version push? This is already part of the automated release procedure, thus we would jump one empty version.

Comment (MiShibari) [inline package.json]: Edited my pull request... hope that's sufficient. I'm very new to github and hope not to mess around to much.

Comment (wzrdtales) [inline package.json]: no worries

**Comment by MiShibari** [comment]
Thank you for your fast support :)

**Pull request merged**